src/towncrier/test/test_build.py

```diff
@@ -500,54 +501,92 @@ def test_no_package_changelog(self):
             ).lstrip(),
         )
 
-    def test_single_file(self):
+    def test_single_file_false(self):
```

Comment:
maybe a better description of the targeted feature

```suggestion
    def test_release_notes_in_separate_files(self):
```